apps/prairielearn/src/tests/activeAccessRestriction.test.ts

```diff
@@ -243,7 +244,7 @@ describe('Exam and homework assessment with active access restriction', function
   );
 
   step('access the exam when it is no longer active', async () => {
-    headers.cookie = 'pl_test_date=2010-01-02T00:01:01Z';
+    headers.cookie = 'pl_test_date=2010-01-10T00:00:01Z';
```

Comment:
Because of timezones, the original date was actually still in the active period. I bumped it forward 10 days to avoid any ambiguity.